### PR TITLE
9806 nested criteria headings

### PR DESCRIPTION
**Pull request by emitche**
- Add visual hierarchy/nesting to loan question headings

### CODE REVIEW

**Review by meltheadorable — COMMENTED** [review]
I think this looks fine codewise, what does this look like in the UI?

**Comment by emitche** [comment]
Edit loan memo criteria
image

Edit loan memo post-analysis
image

**Comment by emitche** [comment]
Print loan memo
9806_tree_of_questions

**Comment by emitche** [comment]
Current print loan memo (without changes in this branch)
9806_tree_of_questions_original

**Comment by emitche** [comment]
Original edit loan memo criteria
9806_edit_loan_memo_criteria_original

**Comment by emitche** [comment]
The size difference of the headings could further be refined.

File: app/assets/stylesheets/admin/loans/_criteria.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.criteria, .post_analysis {
```
[the remark this answers is not in the document]
Comment (emitche) [reply app/assets/stylesheets/admin/loans/_criteria.scss:1]: This error is for an existing CSS class. To change this, we'd need to look into the code and make sure the underscore naming can be easily changed/is not being used for forms in some way that may make a switch more complicated.

**Pull request merged**